ext/excluded_modules.c

```diff
@@ -56,9 +56,9 @@ void ddtrace_excluded_modules_startup() {
         if (module && module->name && module->version && ddtrace_is_excluded_module(module, error)) {
             ddtrace_has_excluded_module = true;
             if (strcmp("xdebug", module->name) == 0) {
-                ddtrace_log_err(error);
+                LOG(Error, error);
             } else {
-                ddtrace_log_debug(error);
+                LOG(Warn, error);
```

Comment:
Why not debug?

Reply:
That else branch is dead code, I don't know why exactly it was written like that.